tests/playwright-test/playwright.ct-react.spec.ts

```diff
@@ -453,7 +453,7 @@ test('should prioritize the vite host config over the baseUrl config', async ({
 
       test('pass component', async ({ page, mount }) => {
         const component = await mount(<Component />);
-        await expect(page).toHaveURL('http://127.0.0.1:8080/');
+        await expect(page).toHaveURL('http://127.0.0.1:8080/index.html');
```

Comment:
This breaks tests that validate the URL. Related to: https://github.com/microsoft/playwright/pull/17767